benchmarks/diffusion_2D/main_arkode.cpp

```diff
@@ -174,10 +172,24 @@ int main(int argc, char* argv[])
 
     // Create linear solver
     SUNLinearSolver LS = NULL;
+    SUNMatrix A = nullptr;
+#if defined(USE_SUPERLU_DIST)
+    // SuperLU-DIST objects
+    SuperMatrix A_super;            // matrix object
+    gridinfo_t grid;                // process grid
+    dLUstruct_t A_lu;               // dLUstruct_t
```

Comment:
Remove unnecessary comments (no need to comment that `dLUstruct_t` is in fact a `dLUstruct_t`.